app/retail/templates/contributor_landing.html

```diff
@@ -85,6 +91,16 @@ <h2 class="join-gitcoin-heading">{% trans "<span class=\"light\">All that's left
     {% include 'shared/rollbar.html' %}
   </body>
   {% include 'shared/analytics.html' %}
-  {% block scripts %}{% endblock %}
+  {% if tech_stack %}
+    <script src="{% static "v2/js/tokens.js" %}"></script>
```

Comment:
i wonder if this should go in its own shared template instead of being coded into the LP?

Reply:
perhaps in `landing/contributor/explorer.html`